docs/source/learn/examples/dimensionality.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Reiterate this is the "implied" situation again?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>A general advice regarding shape, probably worth adding at the top: __Pick prime numbers for dimension lenghts.__</p><p>Then one can always reconstruct how, for example, two variables were concatenated.</p><p>Also makes it easier to pinpoint errors.</p><p> </p><p>In this example, using <code>eye(3)</code>  would make it clear which output dimensions correspond to the support dim.</p><p> </p>

Reply:
<p>I dont quite get what you mean here, though I took a guess and added a new cell to show the limit of where Im losing the understanding</p>

Reply:
<p>What Michael meant is that instead of having a (2, 2) distribution (repeated dimensions), you should use something like (2, 3), so that it's obvious what is base and what is implied/replicated dimensionality. </p><p> </p><p>Your example now needs a size 3 vector mu though. That's why it's failing</p>